File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/address_rsdt/EditWktblRsdtChangeCapsuleDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.address_rsdt;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkCapsuleDto;
import net.seijishikin.jp.normalize.manage.kanrensha.entity.WkTblAddressRsdtChangeEntity;

/**
 * 住居差分ワークテーブル変更Dto
 */
public class EditWktblRsdtChangeCapsuleDto extends FrameworkCapsuleDto //
        implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 住所差分ワークテーブル変更Entity */
    private WkTblAddressRsdtChangeEntity editEntity = new WkTblAddressRsdtChangeEntity();

    /**
     * 住所差分ワークテーブル変更Entityを取得する
     * 
     * @return 住所差分ワークテーブル変更Entity
     */
    public WkTblAddressRsdtChangeEntity getEditEntity() {
        return editEntity;
    }

    /**
     * 住所差分ワークテーブル変更Entityを設定する
     * 
     * @param editEntity 住所差分ワークテーブル変更Entity
     */
    public void setEditEntity(final WkTblAddressRsdtChangeEntity editEntity) {
        this.editEntity = editEntity;
    }

}

File: app/front/src/components/main/dto/address_registory/editWktblRsdtChangeCapsuleDto.ts
import { FrameworkCapsuleDto, type FrameworkCapsuleDtoInterface } from "seijishikin-jp-normalize_common-tool";
import { WkTblAddressRsdtChangeEntity, type WkTblAddressRsdtChangeEntityInterface } from "../../entity/wkTblAddressRsdtChangeEntity";

interface EditWktblRsdtChangeCapsuleDtoInterface extends FrameworkCapsuleDtoInterface {

    /** 住所差分ワークテーブル変更Entity */
    editEntity: WkTblAddressRsdtChangeEntityInterface;

}

class EditWktblRsdtChangeCapsuleDto extends FrameworkCapsuleDto
    implements EditWktblRsdtChangeCapsuleDtoInterface {

    /** 住所差分ワークテーブル変更Entity */
    editEntity: WkTblAddressRsdtChangeEntityInterface;

    constructor() {
        super();

        this.editEntity = new WkTblAddressRsdtChangeEntity();
    }
}

export { type EditWktblRsdtChangeCapsuleDtoInterface, EditWktblRsdtChangeCapsuleDto }
